docker-rancher/server.json

```diff
@@ -1,6 +1,7 @@
 {
   "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
   "contentVersion": "1.0.0.0",
+  "apiProfile": "2019-03-01-hybrid",
```

Comment:
remove this since you're not actually using the profile (each resource has the apiVersion specified)

Reply:
ok